docs/about/neon_ecosystem.md

```diff
@@ -33,7 +30,12 @@ The **Neon EVM program** is a smart contract on the Solana blockchain. The Neon
 
 ## Neon Proxy
 
-Neon Proxy is the "translator', an essential tool for packaging a Neon transaction into a Solana transaction. The proxy provides a containerized solution that's intuitive for [Operators](/docs/operating/overview/introduction) in the Neon ecosystem. It's the Neon Proxy that allows Ethereum dApps to be ported to Neon with virtually no code or configuration changes.
+Neon Proxy is the "translator', an essential tool for packaging a Neon transaction into a Solana transaction. The proxy provides a containerized solution that's intuitive for [Operators](/docs/operating/overview/introduction). It's the Neon Proxy that allows Ethereum dApps to be ported to Neon with minimal reconfiguration.
```

Comment:
We are trying to avoid naming Neon Proxy “translator“ (cause it means changing the original transaction)

Reply:
✅